docs/how-to/submariner-poc-2-gateways-resiliency-walkthrough.md

```diff
@@ -0,0 +1,369 @@
+
+* Local development environment has been set up as per the main README i.e. local env files have been created with AWS credentials & a zone
+
+>**Note:** :exclamation: this walkthrough will setup a zone in your AWS account and make changes to it for DNS purposes
+
+>**Note:** :exclamation: `replace.this` is a place holder that you will need to replace with your own domain
+
+## Installation and Setup
+
+For this walkthrough, we're going to use multiple terminal sessions/windows, all using `multicluster-gateway-controller` as the `pwd`.
+
+Open three windows, which we'll refer to throughout this walkthrough as:
+
+* `T1` (Hub Cluster)
+* `T2` (Where we'll run our controller locally)
+* `T3` (Workloads cluster)
+
+To setup a local instance with submariner, in `T1`, run:
+
+```bash
+make local-setup OCM_SINGLE=true SUBMARINER=true MGC_WORKLOAD_CLUSTERS_COUNT=1
+```
+
+In the hub cluster (`T1`) we are going to label the control plane managed cluster as an Ingress cluster:
+
+```bash
+kubectl label managedcluster kind-mgc-control-plane ingress-cluster=true
+kubectl label managedcluster kind-mgc-workload-1 ingress-cluster=true
+```
+
+Next, in `T1`, create the ManagedClusterSet that uses the ingress label to select clusters:
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: cluster.open-cluster-management.io/v1beta2
+kind: ManagedClusterSet
+metadata:
+  name: gateway-clusters
+spec:
+  clusterSelector:
+    labelSelector: 
+      matchLabels:
+        ingress-cluster: "true"
+    selectorType: LabelSelector
+EOF
+```
+
+Next, in `T1` we need to bind this cluster set to our multi-cluster-gateways namespace so that we can use those clusters to place gateways on:
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: cluster.open-cluster-management.io/v1beta2
+kind: ManagedClusterSetBinding
+metadata:
+  name: gateway-clusters
+  namespace: multi-cluster-gateways
+spec:
+  clusterSet: gateway-clusters
+EOF
+```
+
+### Create a placement for our gateways
+
+In order to place our gateways onto clusters, we need to setup a placement resource. Again, in `T1`, run:
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: cluster.open-cluster-management.io/v1beta1
+kind: Placement
+metadata:
+  name: http-gateway
+  namespace: multi-cluster-gateways
+spec:
+  numberOfClusters: 2
+  clusterSets:
+    - gateway-clusters
+EOF
+```
+
+### Create the gateway class
+ 
+Lastly, we will set up our multi-cluster gateway class. In `T1`, run:
+
+```bash
+kubectl create -f hack/ocm/gatewayclass.yaml
+```
+
+### Start the Gateway Controller
+
+In `T2` run the following to start the Gateway Controller:
+
+```bash
+(export $(cat ./controller-config.env | xargs) && export $(cat ./aws-credentials.env | xargs) && make build-controller install run-controller)
+```
+
+### Create a gateway
+
+We know will create a multi-cluster gateway definition in the hub cluster. In `T1`, run the following: 
+
+**Important**: Make sure to replace `sub.replace.this` with a subdomain of your root domain.
```

Comment:
i think adding the ❗  symbol to important would make it easier to pick out like the notes at the top of the walkthrough

Reply:
Same in other walkthrough as well